index.js

```diff
@@ -4,17 +4,33 @@ const child = require('child_process')
 const eos = require('end-of-stream')
 const wrap = require('spawn-wrapper')
 const path = require('path')
+const shell = require('shelljs')
 const jeopardy = path.resolve(__dirname, 'jeopardy.mp3')
 
+const players = {
+  'afplay': (audioFile) => ['afplay', [ audioFile ]],
+  'mplayer': (audioFile) => ['mplayer', [ audioFile ]],
+  'mpv': (audioFile) => ['mpv', [ '--no-audio-display', audioFile]],
+  'vlc': (audioFile) => ['vlc',[ '-I', 'rc',  audioFile ]]
+}
+
+const findPlayer = function () {
+  for (const playerName of Object.keys(players)) {
+    if (shell.which(playerName)) {
```

Comment:
Would rather use https://www.npmjs.com/package/which over shelljs. Smaller and async.